toolchain/check/context.cpp

```diff
@@ -75,7 +75,7 @@ auto Context::DiagnoseDuplicateName(Parse::Node parse_node,
                     "Duplicate name being declared in the same scope.");
   CARBON_DIAGNOSTIC(NameDeclarationPrevious, Note,
                     "Name is previously declared here.");
-  auto prev_def = semantics_ir_->nodes().Get(prev_def_id);
+  auto prev_def = sem_ir_->nodes().Get(prev_def_id);
```

Comment:
```suggestion
  auto prev_def = nodes().Get(prev_def_id);
```

There are a bunch more of these; looks like the ones of the form `sem_ir_->` didn't get caught, only the ones of the form `sem_ir().`?

Reply:
Fixed `sem_ir_` cases.